apps/judicial-system/web/src/routes/Court/InvestigationRequest/Ruling/Ruling.tsx

```diff
@@ -26,6 +27,8 @@ const Ruling = () => {
     document.title = 'Yfirlit kröfu - Réttarvörslugátt'
   }, [])
 
+  useRulingAutofill(isCaseUpToDate, workingCase)
```

Comment:
I think, instead of having a useRulingAutofill hook we should add the code inside it in the useEffect in this file, where we are already doing some autofilling.